src/System.Security.Cryptography.Cng/tests/ImportExportTests.cs

```diff
@@ -2,8 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Win32.SafeHandles;
-using Test.Cryptography;
+using System.Security.Cryptography;
```

Comment:
This using is redundant, since it's a sub-element of the namespace block to come.